src/replayToMovie.js

```diff
@@ -81,6 +79,46 @@ const p5Inst = new P5(function (p5obj) {
   p5obj.height = HEIGHT;
 });
 
+createNewP5RenderingCanvas();
```

Comment:
Why do we call this statically here (and not keep the reference to the created canvas)?

Reply:
For silly reasons; the Effects constructor uses `p5.rgb` in a couple places, which internally references `p5._renderer`, so we need to have one created before initializing the effects.

I'll add a comment to this effect